boa/src/syntax/ast/node/operator/bin_op/mod.rs

```diff
@@ -86,7 +87,12 @@ impl Executable for BinOp {
                     NumOp::Sub => x.sub(&y, interpreter),
                     NumOp::Mul => x.mul(&y, interpreter),
                     NumOp::Exp => x.pow(&y, interpreter),
-                    NumOp::Div => x.div(&y, interpreter),
+                    NumOp::Div => {
+                        if y == Value::bigint(BigInt::from(0)) {
```

Comment:
Should this be `0.into()`?

Reply:
Is this comment still relevant? (if not you should close it)